code/modules/migrations/ss13/0xx-example.dm

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
probably better to just document this somewhere instead of including the mock file

consider our hack.md